schemas/context/experienceevent.example.1.json

```diff
@@ -88,7 +88,7 @@
     "xdm:geo": {
       "@id": "https://data.adobe.io/entities/geo/tokyo",
       "xdm:countryCode": "JP",
-      "xdm:stateProvince": "JP-13",
+      "xdm:stateProvinceCode": "JP-13",
```

Comment:
Value needs to be changed too. Validation(`maxLength=2`) did not fail because the field name is missing the prefix in the schema.

Reply:
Correcting this.